task/06-conditions-n-loops-tasks.js

```diff
@@ -119,7 +139,13 @@ function isTriangle(a,b,c) {
  *  
  */
 function doRectanglesOverlap(rect1, rect2) {
-    throw new Error('Not implemented');
+    if(rect1.top < rect2.top + rect2.width && rect2.top < rect1.top + rect1.width &&
```

Comment:
You need to return the result of a boolean expression.
The "if, else if" construction is redundant.

Reply:
Fixed.